### ISSUE TITLE
Re-import from a 3rd module results in installation of containerized symbols.

### ISSUE THREAD

**Issue by vrurg** [rakudo/rakudo#2979]
## The Problem

Exporting symbols using `EXPORT` routine results in them being installed in containerized form; i.e. wrapped into a `Scalar`). Under certain circumstances this may result in failing mutli-dispatch due to `derive_dispatch` method trying to act on a `Scalar` instance.

Occasionally similar behavior was observed with `is export` trait.

## Steps to Reproduce

_Foo.pm6_:

```
unit module Foo;

our @foo is export;
sub foo is export { }
```

_Bar.pm6_:

```
use Foo;
our @b;
sub b { }

multi EXPORT {
     return %(
         Foo::EXPORT::ALL::,
         '&bar' => &b,
         '@bar' => @b,
     );
}
```

Script:
```
use Bar;
say @foo.VAR.^name;
say &foo.VAR.^name;
say @bar.VAR.^name;
say &bar.VAR.^name;
```

This would output:

>```
>Scalar
>Scalar
>Scalar
>Scalar
>```

Where expected output would be:

>```
>Array
>Sub
>Array
>Sub
>```

## Environment
* Operating system: macOS
* Compiler version (`perl6 -v`): This is Rakudo version 2019.03.1 built on MoarVM version 2019.03

**Comment by vrurg** [comment]
I have a fix, the ticket is just to have an anchor for tests and PR.

**Comment by vrurg** [comment]
Closed via #2980
